astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -151,51 +131,29 @@ class NDArithmeticMixin(object):
 
     Examples
     --------
+    Using this Mixin with `~astropy.nddata.NDData`:
+
+        >>> from astropy.nddata import NDData, NDArithmeticMixin
+        >>> class NDDataWithMath(NDArithmeticMixin, NDData):
+        ...     pass
+
+    Using it with one operand on an instance::
+
+        >>> ndd = NDDataWithMath(100)
+        >>> ndd.add(20)
+        NDDataWithMath(120)
+
+    Using it with two operand on an instance::
+
+        >>> ndd = NDDataWithMath(100)
+        >>> ndd.divide(1, 5)
```

Comment:
This is probably a silly question (whose answer I could probably google), but is there any benefit to using the instance method (as opposed to the class method) to do arithmetic on two NDData-like objects? If not, could maybe drop this bit.